### ISSUE TITLE
Channels with 90% remote balance can't recieve payments

### ISSUE THREAD

**Issue by salenjak** [Ride-The-Lightning/RTL#639]
I can't send from my Phoenix wallet (also tried with Breez) to RTL wallet. Every payment fails with error message: no route

Was opened a 2m sats channel to one of the top nodes according to BOS Score.
Then was successfuly loop out 1.9m sats to get 1.9m sats on remote balance and 100k on local balance.

Also, I have 2nd channel with 1000 sat on remote, but I still can recieve even payments small as 10 sats or 100 sats.

Does anybody know what can be a reason for not be able to recieve payments although having a channel with plenty of remote capacity?

* Version of `RTL`: 0.10.1-beta
* Version of `lnd`: v0.12.1-beta
* Running on Mynode

**Comment by saubyk** [comment]
Hi @salenjak are of the channels you opened private?

If not, when was the channel opened? If the channel was relatively new, It may take some time for the wallet node to become aware of your channel.

**Comment by salenjak** [comment]
Hi saubyk,

Thanks for helping with my issue.

I have opened both channels on Friday, April 2, 2021. Those are first and only channels I have opened on mynode via RTL wallet.

Opened channel A with 2m sats, loop out 90%, then opened channel B with that looped out amount.
Now have: Channel A (100k/1.9m), Channel B (1.8m/10k)

I have send few transactions to my Phoenix wallet and then tried to pay back to RTL wallet but can't find route. I have even set fees in Phoenix wallet from standard 12sats+0.3% to 100sats+10% and tried to push 1000sats to RTL but fails every time.
Also, have tried to use circular rebalance feature as suggested in this video https://www.youtube.com/watch?v=09tLEZM4MgQ but 
payment fails with "no route" error.

**Comment by salenjak** [comment]
Update:

I have menaged to send 1000sats from Phoenix wallet to RTL but with 110sats fee :(

**Comment by salenjak** [comment]
I have opened new channel with OpenNode and after rebalancing i can send recieve everything with normal fees.

The problem was the first channel I opened with SilentBob node. Althoug I have remote balance this channel for some reason won't  route a payment to my node. Only outgoing transactions.

There is something wrong with this channel because when going in Peers/Channels  -----> View Remote Fee I get error message: "Channel Lookup Failed
Error Code
500
Error Message
Unknown Error"

I don't know is the reason SilentBob  can't route a payment to me on my LND or their side, but I will close this issue now.
